exo-go/src/process_helpers/index.go

```diff
@@ -34,7 +34,9 @@ func RunAndLog(dir string, env []string, log func(string), commandWords ...strin
 	}
 	process := NewProcess(commandWords...)
 	process.SetDir(dir)
-	process.SetEnv(env)
+	if len(env) > 0 {
+		process.SetEnv(env)
+	}
```

Comment:
hey @charlierudolph what do you think about this? I was running into a problem where `terraform` commands couldn't find the proper aws credentials. It seems like if I pass an empty `env` array into this function then the process will be run with a literal empty mapping of env vars. Whereas if we only set the env vars when actual values are present then we avoid overwriting existing env vars?

I think Terraform may be setting my AWS credentials as env vars automatically somehow by reading my ~/.aws/credentials file, and if we pass in an empty env vars map then that will overwrite these credentials? Although I'd expect the order to be check env vars and then check credentials file. Not sure if that makes any sense, but the terraform commands worked after I made this change.